schemas/com.snowplowanalytics.monitoring.loader/alert/jsonschema/1-0-0

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Webhook payload describing an alert coming from loader",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.loader",
+    "name": "alert",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "application": {
```

Comment:
It might be easier to query this if the application name and version were in different fields?

Reply:
Why not. Then `application` field would be just name and `version` would just version?

Reply:
Yeah essentially you would have the app_name and app_version as different fields - but if the current convention is to have them concatenated we can leave it as is - not a strong feeling!